README.md

```diff
@@ -230,3 +230,40 @@ var behavior = {};
 var stats = fetchBuilder(behavior).stats;
 console.log("Hit ratio", stats().cacheHitRatio);
 ```
+
+
+## Timeout
+Example:
+If you know the server response time is 3 seconds or 3000 ms you can configure exp-fetch timeout the following way.
+```
+timeout: {
+    socket: 3500,
+    request: 4000,   
+}
+```
+NOTE: This is are rare case. If have absolute control over responding server and have access to modify the timeout. See examples/timeout.js (to run copy the file to root and run with node). 
```

Comment:
is are

Reply:
I don't understand. Why do you need to configure it to 3500 and 4000 if the response takes 3s?

Reply:
If socket timeout is 3000 ms then you will get a `ESOCKETTIMEOUT`, so the value (socket timeout) needs to be larger than 3s, for instance 3100 ms works fine If I run locally but this may be different if you run the service in another environment.